### PR TITLE
Added unit conversion for inputs.

### PR DESCRIPTION
**Pull request by MichaelClerx**
Seems to work for the initial value version

For the define version there's an issue with initial values not being available, not sure why

### CODE REVIEW

**Comment by MichaelClerx** [comment]
Sorry, draft PR!

File: fc/parsing/actions.py [hunk 1]
```diff
@@ -969,8 +968,6 @@ def _convert_time_if_needed(self):
     def _add_input_variables(self):
         """Ensure requested input variables exist, unless they are optional.
 
-        Note that units conversions are not done yet. If the variable exists in the model, we do nothing.
```
Comment (jonc125) [inline fc/parsing/actions.py]: This will need updating to describe the units conversion (when working).

**Comment by MichaelClerx** [comment]
Needs https://github.com/ModellingWebLab/cellmlmanip/pull/258 to work

File: fc/parsing/actions.py [hunk 2]
```diff
@@ -888,7 +888,8 @@ def modify_model(self, model, units):
         # Convert free variable units
         self._convert_time_if_needed()
 
-        # Ensure inputs exists, adding them if needed, and storing initial values set by user
+        # Ensure inputs exists, adding them if needed, and storing initial values set by user.
```
Comment (jonc125) [inline fc/parsing/actions.py]: exists -> exist :)

**Pull request merged**